docs/users.rst

```diff
@@ -1618,6 +1603,66 @@ Manifests will not be pushed to the application registry for scratch builds,
 isolated builds, or re-builds to prevent unwanted changes
 
 
+Inspecting built image components
+=================================
+
+It is possible to inspect OSBS built image contents from within the image
+container.
+
+In addition to being able to do so with the package manager available in the
+image, if any, e.g., RPM through ``rpm -qa`` to list all the packages installed
+in the image, OSBS also makes sure the following artifacts are shipped within
+the image
+
+Dockerfiles
+-----------
+
+The Dockerfiles used to build the current container image and its parents,
+which is located in the ``/root/buildinfo`` directory.
+
+Note that this is not necessarily the same Dockerfile provided by the user in
+the dist-git repository. OSBS makes changes to the Dockerfile and some of these
+changes may appear in these files whenever relevant. For instance, the FROM
+instruction may show the parent image digest instead of the repository and tag
+information.
+
+Image Content Manifests
+-----------------------
+
+Image Content Manifests are JSON files shipped in OSBS built images with additional
+information on the contents shipped in the image.
+
+The Image Content Manifest file is layer specific, and is located under the
+``/root/buildinfo/content_manifests`` directory. It is named after the image NVR,
+and it is validated against the JSON Schema that defines the `Image Content Manifest`_.
+
+
+Among the data available in the Image Content Manifest file (Check the JSON
+Schema for further information), most important are the ``image_layer_index``,
+which point to the layer that introduced the components listed in that file,
+and:
+
+.. _`Image Content Manifest`: https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/schemas/content_manifest.json
+
+Content Sets
+~~~~~~~~~~~~
+
+The ``content_sets`` field lists the content sets used during the image build
```

Comment:
"the content sets used during the image build"

this isn't exactly truth

because they can have content_sets.yaml BUT not have container.yaml
in that case content sets weren't used, but will be listed here

Reply:
Changed to say it is whatever the user has specified in dist-git for the given platform

Reply:
Should we specify dist-git in upstream docs? I found it in a few other places in the docs as well, but isn't it quite specific to our use case?

Reply:
Good question. I am not sure how well koji deals with non dist-git repositories nowadays (I found only a single reference to the lookaside cache in the code). OSBS upstream does reference it quite often though.

I do agree it is not needed to say dist-git here though. Changed to `git`